### PR TITLE
Change docker container IP detection

### LINKED ISSUE #50: Wrong docker IP detection with docker for Mac
Here https://github.com/zf-fr/zfr-eb-worker/blob/93078f58a54b623dfa6a28503892f4738ccba957/src/Middleware/WorkerMiddleware.php#L157 the docker IP detection seems to be wrong when using docker for Mac, my container IP is `172.18.0.1` so the right regexp could be something like `172.1*.*` maybe...

### PR DESCRIPTION
**Pull request by wysow**
Docker container IP detection is not really working when using docker for Mac.

Fix #50

### CODE REVIEW

**Comment by wysow** [comment]
I just found that the async call to send messages in batch is not really working as the promise is never resolved, fixing this in my second commit.

File: src/MessageQueue/MessageQueue.php [hunk 1]
```diff
@@ -151,7 +151,9 @@ private function doFlush(bool $async)
             }
 
             if ($async) {
-                $this->sqsClient->sendMessageBatchAsync($parameters);
+                $promise = $this->sqsClient->sendMessageBatchAsync($parameters);
+
+                $promise->wait();
```
Comment (bakura10) [inline src/MessageQueue/MessageQueue.php:156]: Not sure to understand here. Isn't the goal of async especially not waiting and be in "fire and forget" mode?

Comment (wysow) [reply src/MessageQueue/MessageQueue.php:156]: Sure but a promise should have a `then` to be launched and do something with the result, here as we do not need to get the result we have to ask the sdk to really send the commands.

Comment (wysow) [reply src/MessageQueue/MessageQueue.php:156]: The commands are sent in parallel in batch of 25 AFAIK.

**Comment by bakura10** [comment]
Makes sense. However the tests are failing now :)

Envoyé de mon iPhone

**Comment by bakura10** [comment]
Just tagged 6.0.2

**Pull request merged**